### PR TITLE
Hs60 win osx dual

### PR DESCRIPTION
**Pull request by mcmadhatter**
## Description

This keycap is a standard Windows 60% ISO keycap by default. It also has an OSX layer that makes sure that the keyboard keycaps (in the windows ISO layout) will create the correct key presses in OSX without having to change the system preferences. For example pressing the [~ #] key will give you a # symbol which usually on ISO OSX  would have to be ALT+3.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/hs60/v2/rules.mk [hunk 1]
```diff
@@ -50,7 +50,7 @@ DFU_ARGS = -d 0483:df11 -a 0 -s 0x08000000:leave
 #   comment out to disable the options.
 #
 BACKLIGHT_ENABLE = no              # Enable keyboard backlight functionality
-BOOTMAGIC_ENABLE = no              # Virtual DIP switch configuration
+BOOTMAGIC_ENABLE = yes             # Virtual DIP switch configuration
```
Comment (drashna) [inline keyboards/hs60/v2/rules.mk]: Could you revert this? 

```suggestion
BOOTMAGIC_ENABLE = no             # Virtual DIP switch configuration
```

If needed, add this to your keymap's `rules.mk`.

Unless @yiancar is okay with this, that is.

**Comment by yiancar** [comment]
As @drashna said, if you require the bootmagic option, please add it in a rules.mk folder for your specific keymap

File: keyboards/hs60/v2/rules.mk (hunk 1 above)
Comment (mcmadhatter) [reply keyboards/hs60/v2/rules.mk]: moved it to the keycap rules.mk

**Comment by yiancar** [comment]
Looks good to me

**Comment by drashna** [comment]
Awesome, thanks!

**Pull request merged**